### PR TITLE
doc: ensure consistency in links.txt

### PR DESCRIPTION
**Pull request by greg-fer**
Fixed the link order based on guidelines.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -95,6 +93,18 @@
 .. _`Project CHIP Protocol Overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/README.md
 .. _`nRF Connect platform overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/docs/guides/nrfconnect_platform_overview.md
 
+.. _`Azure support for Asset Tracker v2`: https://github.com/nrfconnect/sdk-nrf/pull/3950
+
+.. ### Source: github.io
+
+.. _`TinyCBOR`: https://intel.github.io/tinycbor/current/
+
+.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.5.x/docs/device-cloud-protocol/FourKindsOfData.html#timestamping
```
Comment (peknis) [inline doc/nrf/links.txt:102]:
```suggestion
.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.5.x/docs/guides/FourKindsOfData.html#timestamping
```

File: doc/nrf/links.txt [hunk 2]
```diff
@@ -95,6 +93,18 @@
 .. _`Project CHIP Protocol Overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/README.md
 .. _`nRF Connect platform overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/docs/guides/nrfconnect_platform_overview.md
 
+.. _`Azure support for Asset Tracker v2`: https://github.com/nrfconnect/sdk-nrf/pull/3950
```
Comment (peknis) [inline doc/nrf/links.txt:96]: Are you sure you want to link to a pull request?

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/links.txt (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/links.txt:102]: I don't want to touch the link URLs in this PR, just their placement within the file.
JIRA created: https://projecttools.nordicsemi.no/jira/browse/NCSDK-9285

File: doc/nrf/links.txt (hunk 2 above)
Comment (greg-fer) [reply doc/nrf/links.txt:96]: As pointed out by Pekka and Uma, this is addressed in https://github.com/nrfconnect/sdk-nrf/pull/3950

**Review by FrancescoSer — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**